templates/aggkit/aggkit-config.toml

```diff
@@ -519,6 +549,14 @@ InitialBlock = "{{.zkevm_rollup_manager_block_number}}"
 # ------------------------------------------------------------------------------
 BlockFinality = "LatestBlock"
 
+# ------------------------------------------------------------------------------
+# SyncMode denotes should the latest global exit root be determined
+# by querying the global exit root map (which is common way for FEP chains)
+# or the events emitted by sovereign chains (which is a common way for PP chains)
+# Valid values: "FEP", "PP"
+# ------------------------------------------------------------------------------
+# SyncMode = "FEP"
```

Comment:
You can remove this parameter, as it runs by default this way for the `v0.5.0-beta4`. But with `v0.5.0-beta5` (or how ever we decide to name it), it is removed and it is deduced automatically which sync mode we should use, by figuring out the deployed l2 ger manager sc (in case it is `PolygonZkEVMGlobalExitRootv2`, it is "legacy/pessimistic" way of syncing. OTOH if it is `GlobalExitRootManagerL2SovereignChain`, it is the op/fep way of syncing, by tracking the GER insertions and removal events.